### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by StarDustSD** [NateScarlet/auto-derby#340] [Q&A] [answered]
用mumu连了adb，出现了这个，是因为台版的原因吗，还是说我哪里配置错了
![捕获](https://user-images.githubusercontent.com/109811568/182630051-c52c43ee-fe85-4cee-9819-c74d547027ab.PNG)
![捕1获](https://user-images.githubusercontent.com/109811568/182630068-c50e7cdf-45cb-4edd-8bca-18afca36a750.PNG)

**Comment by NateScarlet** [comment, marked as answer]
看FAQ

推荐阅读[《提问的智慧》](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)

**Reply by StarDustSD** [reply]
了解
